app/views/documents/_attachment.html.erb

```diff
@@ -56,14 +56,9 @@
         <%= attachment_attributes(attachment) %>
       <% end %>
     </p>
-    <% if attachment.order_url.present? %>
-      <p>
-        <%= link_to t('attachment.headings.order_a_copy'), attachment.order_url,
+    <% if attachment.is_official_document? %>
+      <%= link_to t('attachment.headings.order_a_copy'), "https://www.gov.uk/guidance/how-to-buy-printed-copies-of-official-documents",
```

Comment:
Sorry ignorant drive-by comment. Should this always be www.gov.uk or should we use Plek to have this be environment specific?

Reply:
Good spot! 🤦‍♂

Reply:
poop, yeah fair play, I'll raise a pr to fix this